tools/azsdk-cli/Azure.Sdk.Tools.Cli/Commands/SharedCommandGroups.cs

```diff
@@ -0,0 +1,41 @@
+using Azure.Sdk.Tools.Cli.Contract;
+using System.CommandLine;
+using System.CommandLine.Parsing;
+
+
+namespace Azure.Sdk.Tools.Cli.Commands
+{
+    /// <summary>
+    /// These should be referenced in the CommandHierarchy array for each command if necessary.
+    /// Every access returns new instances so you never reuse the same Option or List when building up the command tree.
+    /// </summary>
+    public static class SharedCommandGroups
+    {
+        public static readonly Option<string> CheckThisOption = new Option<string>(
+            aliases: new[] { "--meep", "-m" },
+            description: "This should be settable anywhere in the assigned command",
+            getDefaultValue: () => "oops"
+        )
+        {
+            IsRequired = false
+        };
+
+        public static readonly CommandGroup AzurePipelines = new CommandGroup(
```

Comment:
<img width="833" alt="image" src="https://github.com/user-attachments/assets/05e5d76f-e9df-4861-9455-57dd3319549e" />

Code that was here before I updated to be ready for review:

```c#
public static readonly Option<string> CheckThisOption = new Option<string>(
            aliases: new[] { "--meep", "-m" },
            description: "This should be settable anywhere in the assigned command",
            getDefaultValue: () => "oops"
        )
        {
            IsRequired = false
        };

        public static readonly CommandGroup AzurePipelines = new CommandGroup(
            Verb: "azp",
            Description: "Azure Pipelines Tool",
            Options: new List<Option> { CheckThisOption }
        );